packages/manager/modules/sms/src/sms/order/telecom-sms-order-service.html

```diff
@@ -197,10 +197,9 @@
                 <div class="mb-3" data-ng-if="$ctrl.prices.url">
                     <span data-translate="sms_order_bc"></span>
                     <a
-                        target="_blank"
                         rel="noopener"
                         title="{{:: 'sms_order_order' | translate}}"
-                        data-ng-href="{{$ctrl.prices.url}}"
+                        data-ng-click="$ctrl.goToDisplayOrder()"
```

Comment:
I think you can both use `ng-click` and `ng-href` on the `a` element to avoid having to set `$window.location`. Also why not use the `track-on` directives provided by the `ng-at-internet` module, it might avoid creating a function